Project/src/main/java/app/ReviewerController.java

```diff
@@ -21,35 +33,53 @@
      */
     @GetMapping("/reviewer")
     public String getReviewer(Model model) {
-        model.addAttribute("articles", articleRepository.findByUsersOrderByDateAsc(super.getUser()));
+        model.addAttribute("articles", articleRepository.findByUsersOrderByReviewDueDateAsc(super.getUser()));
         return "reviewer";
     }
 
     /***
-     * Gets the information about a single article
-     * @param model The model to return the files
-     * @param id The id of the article
-     * @return The review view
+     * Uploads a file to a review
+     * @param file The Multipart file that got uploaded
+     * @return
+     * @throws IOException
      */
-    @GetMapping("/review/{id}")
-    public String getReviewer(Model model, @PathVariable String id) {
-        model.addAttribute("article", articleRepository.findById(Long.parseLong(id)).get());
-        return "review";
+    @PostMapping("/review")
+    public String uploadFile(@RequestParam String articleId, @RequestParam("file") MultipartFile file) throws IOException {
```

Comment:
ArticleId should be a url path variable because it's required in order to post the review.